src/main/java/piglatin/PigLatinConverter.java

```diff
@@ -46,15 +64,38 @@ public static String convertStringToPigLatin(String inputString) {
 			output += convertWordToPigLatin(word) + " ";			
 		}
 		
-		return output.substring(0, output.length()-1);
+		return output.trim();
 	}
 	
-	
-	public static void main(String[] args) {
-		String input = getInputFromCommand();
+	public static List<String> convertStringFromPigLatin(String inputString) {	
+		String[] words = inputString.split(" ");		
+		List<String> outputs = new ArrayList<String>();		
+		outputs.add("");
+		
+		for (String word: words) {
```

Comment:
The loop beginning at row 75 creates a list of every possible combination of all candidates for every word, if I understand it correctly. Sounds like that could be quite a large amount of combinations in the end.

I found it wasn't easy to see what this did at a glance, maybe some more commenting would be nice?